cmd/telemetrygen/internal/traces/worker.go

```diff
@@ -80,6 +80,9 @@ func (w worker) simulateTraces(telemetryAttributes []attribute.KeyValue) {
 		}
 
 		opt := trace.WithTimestamp(time.Now().Add(fakeSpanDuration))
+		w.logger.Info("Trace", zap.String("traceId", sp.SpanContext().TraceID().String()))
+		w.logger.Info("Parent Span", zap.String("spanId", sp.SpanContext().SpanID().String()))
+		w.logger.Info("Child Span", zap.String("spanId", child.SpanContext().SpanID().String()))
```

Comment:
I don't think we should log this for every trace. Telemetrygen is meant to be used for load testing, where you may have thousands of traces reported per second. In this situation we should not log each trace.

What is the use case for this?

Reply:
We use telemetrygen to send the trace to test if the otelgen configuration file has been properly configured. For instance, when sending a trace from a POD in K8s to CloudRun in GCP, sometimes GCP does not have the trace. Therefore, we need to locate the trace by tracking the trace ID. Additionally, the default output is in debug mode, which prints out hundreds of lines. Having a few more lines here doesn't matter, in my opinion.

Reply:
> For instance, when sending a trace from a POD in K8s to CloudRun that running the otelgen receiver/exporter in GCP, sometimes GCP does not have the trace. Therefore, we need to locate the trace by tracking the trace ID.

Are you sending directly to an OTLP endpoint, or are you using the OpenTelemetry Collector? If using the Collector, would the [debug exporter][1] or the [file exporter][2] help?

> Additionally, the default output is in debug mode, which prints out hundreds of lines. 

Not sure what you mean by this. could you elaborate? The current logging does not output anything per trace.

> Having a few more lines here doesn't matter, in my opinion.

II would not want to make the problem worse just because it is bad right now :smile: 

I am not against having this if it's useful, but it should be opt-in, and I would like to use the OTel Go SDK stdout exporter if we go down this route.

[1]: https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/debugexporter
[2]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter#file-exporter

Reply:
> Not sure what you mean by this. could you elaborate? 

The info below is quite useless to me and I'd like to avoid in the ouput.
The `SkipSettingGRPCLogger` is true in the test case, howerever don't see a option to disable that in cli. So it's `false `when I run telemetry locally.
<img width="1644" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/84754386/f8518283-2489-4de7-b98e-052eb6c0d481">

So if I set it to true and compile again with trace id enabled , this is what I am expecting: just few lines with trace ID in the output.
<img width="1514" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/84754386/9e26d19d-d5d7-4ed9-a016-3c359081ee48">

So, back to my use case. It doesn't matter which exporter the OpenTelemetry Collector is using. My objective is straightforward: I want to run telemetrygen on my test pod or my local Mac to confirm that the traces are reaching the end system, such as Datadog or Dynatrace. The easiest way to locate the test trace I sent to the end system is by searching for the trace ID. Without the trace ID, I have no way of knowing whether my trace has been successfully sent to the end system. I am not using telemetrygen for performance testing; my goal is simply to verify the network connection between my test pod and the end system.

The traffic flow will be like this:

Test POD in Kubenetes -> OTELCOL POD -> PROXY -> END SYSTEM (Dynatrace/GCP or Datadog)
sometime the proxy settings or some other FW rules blocking my traffic on port 4317. Using telemetry can quickly help me to verify the connection, and that's why I need Trace ID.

Perhaps my use case isn't common. I might submit another pull request with a proper solution to disable/enable the verbose logs, so I'll close this one.
@mx-psi

Reply:
> The info below is quite useless to me and I'd like to avoid in the ouput.

Right, feel free to file an issue about this if you think we should change it. We have discussed removing it in the past, on the last version we bumped it to warn (see #27079), but based on your logs and my own tests I am not sure that worked.

Still, these are not logs that happen *per trace*, which is the real issue I am trying to get at: one log per trace is not sustainable for load testing use cases.

> My objective is straightforward: I want to run telemetrygen on my test pod or my local Mac to confirm that the traces are reaching the end system, such as Datadog or Dynatrace. The easiest way to locate the test trace I sent to the end system is by searching for the trace ID. Without the trace ID, I have no way of knowing whether my trace has been successfully sent to the end system

Right, I think that is fair, I still don't understand how having the logging built in on telemetrygen instead of using one of these exporters is better, but I am not opposed to adding this kind of logging to telemetrygen so long as:
1. It is disabled by default
2. We don't reinvent the wheel and instead use an existing solution (e.g. the [stdouttrace](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/stdout/stdouttrace)) exporter)